addons/point_of_sale/static/src/app/screens/partner_list/partner_list.js

```diff
@@ -121,6 +121,18 @@ export class PartnerList extends Component {
                 return (a.name || "").localeCompare(b.name || "");
             });
         }
+
+        for(const partner of res) {
+            if(!partner.address) {
+                partner.address =
+                    (partner.street ? partner.street + ", " : "") +
+                    (partner.zip ? partner.zip + ", " : "") +
+                    (partner.city ? partner.city + ", " : "") +
+                    (partner.state_id ? partner.state_id.name + ", " : "") +
+                    (partner.country_id ? partner.country_id.name : "");
+            }
+        }
```

Comment:
why don't we just use `contact_address` instead ?

Reply:
`contact_address` includes the company name, but maybe it is acceptable to have it?
![image](https://github.com/user-attachments/assets/1732b653-d43b-4030-835d-46dd3a881301)

Reply:
i would say that it's fine